product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/02_connect_on_premises.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Connecting from an On-Premises Network
+---
+
+
+Sometimes your applications are deployed on your on-premises
+environments, or on other Cloud providers instead Azure, but they also
+want to consume private EDB Cloud services from these premises via a
+secure approach. In these cases, you can set up a secure tunnel
```

Comment:
I think we need to talk about motivations here. This is only marginally more secure, and the customer is jumping through a lot of hoops when they can just use the public endpoints that we make available.

@zozowell: Would you be okay with not having this page in the docs at all?

Reply:
@ba-edb , which do you prefer:

- Not having a this page in the docs at all
- Keep the page along with the motivations mentioned by Zane as follws:
 
 If your applications are deployed on your on-premises environments, or on other Cloud providers instead Azure, you can set up a secure tunnel between your on premises network and the Azure virtual network. Setting up a secure tunnel enables you to connect remotely or from your organization's staging or production environment.

Reply:
Alright, let's keep it. I'll take another review pass through it.